transforms/unquote-properties.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Removes quotes from object properties whose keys are strings which are valid
+ * identifiers.
+ */
+module.exports = (file, api, options) => {
+  const j = api.jscodeshift;
+  const printOptions = options.printOptions || {quote: 'single'};
+  const root = j(file.source);
+
+  const isValidIdentifierNameForProperty = (name) => {
+    try {
+      new Function(`x = {${name}: 1}`)(); //eslint-disable-line no-new-func
```

Comment:
hah. Not bad indeed. Feels a bit hacky because you can also write a regex to detect whether the name is valid but I'll accept this tiny sample of meta-evaluation. Can you remove the `x = ` and instead do `({${name}: 1})`?